geniza/geniza.py

```diff
@@ -94,3 +94,21 @@ def detect_language(self, text: str) -> dict:
             raise ValueError("You must supply text from which to detect the language.")
 
         return self._client.post('detectors/language', {'text': text})
+
+    def message_summary(self, message: str, word_count=None) -> dict:
```

Comment:
I'm on the fence about this but do you think we should add caching to methods like this?

Reply:
That is a good question.  I am not sure what I think about that either.  IF we did implement caching here, I would recommend a separate pull request.  Let's keep this one clean.